ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -325,16 +138,6 @@ server {
         }
     }
 
-    location /robots.txt {
-        alias /home/www/nodejs/static/robots.txt;
-        default_type text/plain;
-    }
-
-    location /.well-known/security.txt {
```

Comment:
This no longer works/is being served

Reply:
This file doesn't really exist.

Reply:
It did in that location before, and is now located https://github.com/nodejs/nodejs.org/blob/main/public/security.txt

Reply:
No, I think you're confusing things. There was never a well-known/security.txt (https://github.com/nodejs/nodejs.org/tree/fb666ed663e4d0511beced38cf9505a688bde898/static) (ref before next migration).

Or are you trying to say that /.well-known/security.txt should redirect to /security.txt?

Reply:
Oh, I see what you're talking about.

Reply:
This redirect was serving the file in `static/security.txt` through the `.well-known` address, which is per it's spec https://securitytxt.org/

Reply:
https://github.com/nodejs/build/pull/3225

Reply:
> This redirect was serving the file in static/security.txt through the .well-known address, which is per it's spec [securitytxt.org](https://securitytxt.org/)

Aware of that spec, my bad, I just switched these two 👀